app/src/Containers/HomeContainer.tsx

```diff
@@ -36,16 +36,16 @@ const ProjectList: FC<ProjectProps> = ({ data }) => {
 }
 
 const HomeContainer = () => {
-  const [fetchAll, { data: projects }] = useLazyFetchAllQuery()
+  const [fetchAll, { data: listData }] = useLazyFetchAllQuery()
 
   useEffect(() => {
     fetchAll('')
   }, [fetchAll])
 
-  if (projects) {
+  if (listData) {
     return (
       <Theme>
-        <ProjectList data={projects} />
+        <ProjectList data={listData} />
```

Comment:
What was the reason for renaming this? If it was to make it clear that this is returning a list of projects, it might be better named something like `projectListData`?

Reply:
Just tested it and it works when I change it back! I changed it to the final result in the backend because the data wasn't coming.. However, I think the real solution was the changes made in 'fetchAll'.
I've changed it back, thanks Jamie well spotted.